superset/sql_parse.py

```diff
@@ -102,7 +102,7 @@
     "clickhouse": Dialects.CLICKHOUSE,
     "clickhousedb": Dialects.CLICKHOUSE,
     "cockroachdb": Dialects.POSTGRES,
-    "couchbasedb": Dialects.MYSQL,
+    "couchbase": Dialects.MYSQL,
```

Comment:
This would entail a breaking change, and would break every Couchbase connection created by existing users. At minimum we would need to create an alias for the old dialect name to ensure old connections still work.

Reply:
I have corrected this in new patch. I am adding alias to old name also. I am seeing all checks passed too.

Reply:
> I am adding alias to old name also.

I don't see the alias here - can you check how the alias is implemented in the Postgres spec?

Reply:
My mistake I thought alias means updating sql_parse , I saw oceanbase and postgres db_engine_specs and similar to that I have added `engine_aliases = {"couchbase", "couchbasedb"}` to db_engine_spec/couchbase.py .